examples_tests/35.TAA/main.cpp

```diff
@@ -333,6 +351,7 @@ int main()
         // AA PASS
         if (g_FrameNum > 0u) //first frame is without AA
         {
+            TAAcb->setJitterOffset(JITTER_OFFSETS[g_FrameNum%JITTER_OFFSET_CNT]);
```

Comment:
does your TAA shader also know the previous frame's jitter offset so it can properly look things up in history buffer?